python/taichi/lang/matrix.py

```diff
@@ -1476,21 +1479,11 @@ def __call__(self, *args):
             elif isinstance(x, np.ndarray):
                 entries += list(x.ravel())
             elif isinstance(x, Matrix):
-                entries += x.entries
+                entries += [x(i, j) for j in range(x.m) for i in range(x.n)]
```

Comment:
Why do we change this?

Reply:
Ah just realized `Matrix::entries` is already flattened. Thanks!